package.json

```diff
@@ -86,7 +86,7 @@
     "react-native-webrtc": "github:mattermost/react-native-webrtc",
     "react-native-webview": "11.18.2",
     "react-syntax-highlighter": "15.5.0",
-    "readable-stream": "3.6.0",
+    "readable-stream": "4.1.0",
```

Comment:
it looks like this version is preventing hermes from building the js bundle as there must be something in the newest version that the Hermes engine still does not support :S .. @cpoile I think we need to change this back to 3.6.0, sorry for the inconvenience

Reply:
No problem at all, fixed. 👍